.circleci/config.yml

```diff
@@ -67,15 +67,9 @@ jobs:
       - run:
           name: Test - Definition Lists
           command: npm run test:definition-lists
-      - run:
-          name: Test - Hard Breaks
-          command: npm run test:hard-breaks
       - run:
           name: Test - Non-Breaking Spaces
           command: npm run test:non-breaking-spaces
-      - run:
```

Comment:
This is removing the tests for paragraph justification. Let's keep those here just to make sure we are still passing all tests before merging.

Reply:
I had a stray commit here, I'll put it back - the justifications were done first and expected to get merged first so I didn't bother when it happened. :)